static/js/ConnectionsPanel.jsx

```diff
@@ -184,7 +184,19 @@ class ConnectionsPanel extends Component {
     return !prevRefs.compare(nextRefs);
   }
   sectionRef() {
-    return Sefaria.sectionRef(Sefaria.humanRef(this.props.srefs)) || this.props.srefs;
+    const humanRefForm = Sefaria.humanRef(this.props.srefs);
```

Comment:
The function sectionRef exists in ConnectionsPanel because sectionRef in sefaria.js returns null when the input is not found whereas in ConnectionsPanel the current behavior is to default to this.props.srefs if the ref is not found.  Based on your point here, I modified the Sefaria.sectionRef to accept a parameter, `deriveIfNotFound`, so that if it's not found in the cache it won't return null but will try to derive the sectionRef from the input.  I modified the ConnectionsPanel's sectionRef function to be almost exactly as it was originally (as it is now on master), except that it passes a true value as `deriveIfNotFound` to the sefaria.js sectionRef.